recipes/tree-sitter-yaml/recipe.yaml

```diff
@@ -0,0 +1,65 @@
+context:
+  name: tree-sitter-yaml
+  version: "0.7.0"
+  python_min: 3.9
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/tree-sitter-grammars/tree-sitter-yaml/archive/refs/tags/v0.7.0.tar.gz
+  sha256: 8182760587f14d5131161dee3605613ccebe86062909f0879edf63b4bdd99d44
+
+build:
+  script:
+    - if: unix
+      then:
+        - ${{ PYTHON }} -m pip install . --no-deps --ignore-installed --prefix=${PREFIX} -vv
```

Comment:
You should not need the `--prefix` here. If you omit it, you don't need to differentiate between windows and unix.

Reply:
Cool, done.